zokrates_stdlib/stdlib/utils/pack/bool/nonStrictUnpack256.zok

```diff
@@ -1,11 +1,11 @@
-#pragma curve bn128
-
 import "./unpack_unchecked";
 
 // Unpack a field element as 256 big-endian bits
 // Note: uniqueness of the output is not guaranteed
 // For example, `0` can map to `[0, 0, ..., 0]` or to `bits(p)`
-def main(field i) -> bool[256] {
-    bool[254] b = unpack_unchecked(i);
-    return [false, false, ...b];
+def main(field i, u32 bit_size) -> bool[256] {
```

Comment:
TODO: After a second look, I figure this function should have better documentation

cc @alvaro-alonso

Reply:
@dark64 good point!

What about:

```rust
import "./unpack_unchecked";

// Unpack a field element as 256 big-endian bits.
// Since the size of p can differ among different fields, 
// p_size has to be given to derive the correct padding.
// For example, for bn128 |p|=254, for bls12-381 |p|=255.
// Note: uniqueness of the output is not guaranteed
// For example, `0` can map to `[0, 0, ..., 0]` or to `bits(p)`
def main(field i, u32 p_size) -> bool[256] {
    assert(p_size <= 256);
    u32 padding_size = 256 - p_size;
    bool[p_size] b = unpack_unchecked(i);
    return [...[false; padding_size], ...b];
}
```

I can not push the changes to the branch. And I think a PR is overkill 💀